src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php

```diff
@@ -81,6 +94,65 @@ public function configureViews(ViewCollection $viewCollection): void
                 ])
                 ->addToolbarActions([])
         );
+
+        if ($viewCollection->has(PageAdmin::EDIT_FORM_VIEW)) {
+            $viewCollection->add(
+                $this->viewBuilderFactory
+                    ->createResourceTabViewBuilder(static::EDIT_FORM_ACTIVITY_VERSION_TAB_VIEW, '/history')
```

Comment:
think i would just call it `/activity` and use `activity/activity` and `activity/versions` later on. that way we would not introduce a new term `history`

```suggestion
                    ->createResourceTabViewBuilder(static::EDIT_FORM_ACTIVITY_VERSION_TAB_VIEW, '/activity')
```